tests/misc/test_access_links.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import subprocess
+import hashlib
+
+# This test is designed to verify the accessibility of the XOA deployment script
+#
+# Requirements:
+# - an XCP-ng host with xcp-ng-release >= 8.3.0.29
+
+@pytest.mark.parametrize("command", ["curl -fsSL", "wget -qO-"])
+@pytest.mark.parametrize("url", [
+    "https://xoa.io/deploy",
+    "https://updates.xcp-ng.org/trace",
+    "https://repo.vates.tech/README.txt"
+])
+def test_xoa_deploy(host, command, url):
+    """
+    Verifies that the specified URL responds correctly via the specified command
+    and compares the checksum of the downloaded content between local and remote.
+    """
+    COMMAND = f"{command} '{url}'"
+
+    # Download
+    local_result = subprocess.run(
+        COMMAND,
+        shell=True,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        text=True,
+    )
```

Comment:
We have `lib.commands.local_cmd()` to wrap `subprocess` and ensure the command we run appears in the log